CoreDataAccessView/release/modules/ext/docs/CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/extract-words-from-text.md

```diff
@@ -2,6 +2,10 @@
 
 ## Extract Words from a specified transaction attribute
 
+<br />
+<img src="../ext/docs/CoreDataAccessView/src/au/gov/asd/tac/constellation/views/dataaccess/resources/ExtractWordsFromText.png" alt="Extract Words From Text" />
+<br />
```

Comment:
I think for this screenshot, it might be good to have the whole list of parameters visible rather than it being cut-off (might mean that the top needs to be cropped slightly)